src/content/docs/analyzer/plugins.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Analyzer Plugins
+description: An overview of how to use Biome's Analyzer Plugins
+---
+
+:::note
+Plugins are currently only available in the [2.0 beta release](/blog/biome-v2-0-beta).
+:::
+
+Biome Analyzer supports [GritQL](/reference/gritql/) plugins. Currently, these
+plugins allow you to match specific code patterns and register customized
+diagnostic messages for them.
+
+An example of a plugin is:
+
+```grit
+`$fn($args)` where {
+    $fn <: `Object.assign`,
+    register_diagnostic(
+        span = $fn,
+        message = "Prefer object spread instead of `Object.assign()`"
+    )
+}
+```
+
+You can put a GritQL snippet in a file anywhere in your project, but be mindful
+you use the `.grit` extension. Then, you can simply enable it as a plugin with
+the following configuration:
+
+```
+{
+    "plugins": ["./path-to-plugin.grit"]
+}
+```
+
+The plugin will now be enabled on all files the analyzer runs on.
```

Comment:
This assumes that plugins runs in all languages supported by Biome. Is that the case? If so, we should specify it and maybe add a link to `/reference/language-support`

Reply:
Excellent point! Currently it's only CSS  and JS only indeed.